community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstractionTest.java

```diff
@@ -39,7 +39,7 @@
 import org.neo4j.io.fs.OpenMode;
 import org.neo4j.io.fs.StoreChannel;
 
-import static java.nio.ByteBuffer.allocateDirect;
+import static java.nio.ByteBuffer.allocate;
```

Comment:
When you allocate direct buffer you need to manage it yourself including de-allocating it to free that memory that was never done first of all. When you do that you want to put some data outside of heap, for example, to avoid GC to spend its time on it. And in this case, its only additional burden that does not bring any gain at all.

Reply:
Aah, yes of course! The de-allocation. Thanks for explaining ☺️